docs/docsite/rst/playbook_guide/playbooks_module_defaults.rst

```diff
@@ -102,31 +102,54 @@ Setting a default AWS region for specific EC2-related modules.
 Module defaults groups
 ----------------------
 
-.. versionadded:: 2.7
-
-Ansible 2.7 adds a preview-status feature to group together modules that share common sets of parameters. This makes it easier to author playbooks making heavy use of API-based modules such as cloud modules.
-
-+---------+---------------------------+-----------------+
-| Group   | Purpose                   | Ansible Version |
-+=========+===========================+=================+
-| aws     | Amazon Web Services       | 2.7             |
-+---------+---------------------------+-----------------+
-| azure   | Azure                     | 2.7             |
-+---------+---------------------------+-----------------+
-| gcp     | Google Cloud Platform     | 2.7             |
-+---------+---------------------------+-----------------+
-| k8s     | Kubernetes                | 2.8             |
-+---------+---------------------------+-----------------+
-| os      | OpenStack                 | 2.8             |
-+---------+---------------------------+-----------------+
-| acme    | ACME                      | 2.10            |
-+---------+---------------------------+-----------------+
-| docker* | Docker                    | 2.10            |
-+---------+---------------------------+-----------------+
-| ovirt   | oVirt                     | 2.10            |
-+---------+---------------------------+-----------------+
-| vmware  | VMware                    | 2.10            |
-+---------+---------------------------+-----------------+
+Collections can define groups in the ``meta/runtime.yml`` file. ``module_defaults`` does not take the ``collections`` keyword into account, so the fully qualified group name must be used for new groups in ``module_defaults``.
```

Comment:
I would add some text that explains what module default groups actually are. Maybe like this:
```suggestion
Module default groups allow to provide common parameters to groups of modules that belong together. Collections can define such groups in their ``meta/runtime.yml`` file.

.. note::
    ``module_defaults`` does not take the ``collections`` keyword into account, so the fully qualified group name must be used for new groups in ``module_defaults``.
```
I've also moved the part on the `collections` keyword to a note.